torch/serialization.py

```diff
@@ -561,14 +562,16 @@ def persistent_load(saved_id):
             # if not a tarfile, reset file offset and proceed
             f.seek(0)
 
-    magic_number = pickle_module.load(f, **pickle_load_args)
-    if magic_number != MAGIC_NUMBER:
-        raise RuntimeError("Invalid magic number; corrupt file?")
-    protocol_version = pickle_module.load(f, **pickle_load_args)
-    if protocol_version != PROTOCOL_VERSION:
-        raise RuntimeError("Invalid protocol version: %s" % protocol_version)
+    if not getattr(pickle_module, '_unpickle_skip_metadata', False):
```

Comment:
todo: I'm going to remove this flag and have the jit pickler just serialize all this metadata as well so there'll be no difference between `torch.save` in jit/Python execution

Reply:
yes, it would be best if loading torch.load just worked with torch.save from the JIT without extra flags